src/feature_selection.py

```diff
@@ -54,12 +58,12 @@ def feature_selection(df):
         'Fireplaces', 'PoolArea', 'PavedDrive', 'GarageType', 'GarageQual',
         'GarageYrBlt', 'GarageFinish', 'GarageCond', 'GarageArea',
         'WoodDeckSF', 'OpenPorchSF', 'EnclosedPorch', '3SsnPorch',
-        'ScreenPorch', 'FireplaceQu', 'PoolQC',
-        'BsmtCond', 'BsmtExposure', 'BsmtFinType1', 'BsmtFinSF1',
+        'ScreenPorch', 'FireplaceQu', 'PoolQC', 'LotFrontage', 'LotConfig',
+        'BsmtCond', 'BsmtExposure', 'BsmtFinType1', 'BsmtFinSF1', 'LandSlope',
         'BsmtFinType2', 'BsmtFinSF2', 'BsmtUnfSF', 'TotalBsmtSF',
-        'RoofStyle', 'Exterior1st', 'Exterior2nd',
+        'RoofStyle', 'Exterior1st', 'Exterior2nd', 'LotShape', 'LandContour',
         'BldgType', 'Functional', 'MoSold', 'MiscVal', 'LowQualFinSF',
-        'KitchenAbvGr'
+        'KitchenAbvGr', 'BsmtFullBath', 'BsmtHalfBath', 'FullBath', 'HalfBath'
```

Comment:
I don't remember why we droped the bath variables, can you explain that?

Reply:
We dropped them because we created a feature called "TotalBath" that consists on the sum of them.